### PR TITLE
removed school board and associated grade

### PR DESCRIPTION
**Pull request by colinstrancd2l**
Ticket [US87152](https://rally1.rallydev.com/#/95185052972d/detail/userstory/122671160612)

### CODE REVIEW

File: d2l-user-tile.html [hunk 1]
```diff
@@ -89,13 +90,6 @@
 				<p class="user-tile-name" hidden$="[[!_hideNamePlaceholder(placeholders, name)]]">[[name]]</p>
 				<p class="user-tile-name text-placeholder" hidden$="[[_hideNamePlaceholder(placeholders, name)]]">&nbsp;</p>
 
-				<p class="user-tile-school" hidden$="[[!_hideSchoolPlaceholder(placeholders, school, grade)]]">
```
Comment (ryantmer) [inline d2l-user-tile.html]: Looks like this means the `grade`/`school` properties can now be removed, as well as `_hideSchoolPlaceholder` and the now-unused CSS classes :)

**Comment by colinstrancd2l** [comment]
I will say, it pains me to straight up remove all the references that ideally are going to need to be stuffed back in later on. Would it be better to just remove their usages so the framework is still there for the future?

**Comment by ryantmer** [comment]
@colinstrancd2l Could, but dead code usually isn't great. In addition, can always get it back via git! (or just this PR 😄)

**Comment by colinstrancd2l** [comment]
@ryantmer But that would imply someone remembers this ticket and has the patience to find it, some months down the line. But still, definitely fair enough!

**Comment by ryantmer** [comment]
Yeah true - although git blame can also help with such things :)

**Comment by colinstrancd2l** [comment]
Very true! 👍

File: demo/index.html [hunk 2]
```diff
@@ -53,16 +51,6 @@
...
-
-		setTimeout(function() {
-			slowTile.grade = 'Grade 50';
-			placeholderTile.grade = 'Grade 51';
-		}, 4000);
-
-		setTimeout(function() {
 			slowTile.background = 'abe.jpg';
 			placeholderTile.background = 'joan.jpg';
 		}, 5000);
```
Comment (ryantmer) [inline demo/index.html]: The nittiest of nits, but this could change to 3000 now, so that it happens one second after the names get filled in.

**Comment by ryantmer** [comment]
Erm... I have those dependencies backwards. But, same idea - the `user-profile-behavior` change isn't required for this, but that change would breaks things here. So, merge + release this and `user-switcher`, then I can re-check with the `user-profile-behavior` changes (was seeing a bit of weirdness, but I suspect it was a bower link issue)

**Comment by colinstrancd2l** [comment]
Should I be doing any version bumps to this or `user-switcher`?

**Comment by ryantmer** [comment]
In this case, they'd have to be follow-ups - you could:

1. merge + release this and `user-switcher`
2. merge + release `user-profile-behavior`
3. Bump the `user-profile-behavior` version used by this and `user-switcher`

The last one is fairly optional - floating values should grab the most recent on the next merge anyway.

**Comment by colinstrancd2l** [comment]
Awesome, thanks for the summary!

**Pull request merged**